src/Uno.UI/UI/Xaml/Media/Brush.cs

```diff
@@ -38,6 +83,7 @@ protected virtual void OnOpacityChanged(double oldValue, double newValue)
 
 		#endregion
 
+		// TODO: InvalidateRender when Transform/RelativeTransform changes, or their inner properties.
```

Comment:
Please create an issue for this TODO

Reply:
This TODO is actually already done above, I think.

But now revising this part of the code again, it seems that the brush will now have a reference to the transforms, and I'm not sure the transforms can leak. We need to verify that very well.

Reply:
Double thinking, I think no leak concerns here. If a given transform is the current value of `Transform`/`RelativeTransform`, then that is owned by the brush anyway, and if the value changed, we unsubscribe from the event.

So it should be good I think.